fakestorage/upload.go

```diff
@@ -630,6 +630,10 @@ func parseContentRange(r string) (parsed contentRange, err error) {
 	return parsed, nil
 }
 
+func (s *Server) deleteResumableUpload(r *http.Request) jsonResponse {
+	return jsonResponse{status: 499}
+}
```

Comment:
Ideally, if a non-existent session URI is specified, the server should return a 404, and a PUT request for a deleted session URI should respond with 499. However, since the upload process did not seem to specifically check whether the session URI actually exists, I decided to follow that approach and not pursue this level of reality.